Removed the jQuery dependency

sprt opened the pull request:
Hey guys,

The Kippt button is now jQuery-free. :heart:

(I also cleaned up the test page a bit and of course updated _README.md_.)

sprt committed:
Remove the jQuery dependency, fixing #1

sprt committed:
Create a text node instead of using `.innerHTML`

jorilallo commented:
Thanks so much! I'll take a look at this little bit later but it looked good and simple.

TODO for myself, remove the one console.log debug line

sprt committed:
Remove the `console.log()` statement

sprt commented:
Oops! That's what I get for committing too fast. Fixed.

sprt committed:
Simplify the embedding script

sprt committed:
Clean up

sprt committed:
Remove the intertag spaces in the HTML embedding code

sprt committed:
Make the Kippt button use CSS only

sprt committed:
Trailing newlines everywhere

sprt committed:
Variable was previously declared but not used where we could

sprt committed:
Specify a real font

sprt committed:
Add a missing blank line

sprt committed:
Fix a bug that inserts `undefined` values in the popup's URL

For example, if one didn't specified `data-via`, the popup's URL would still
include `via=undefined`.

Also, simplify the whole URL construction process.

sprt committed:
Remove useless CSS properties

sprt committed:
Fix an alignment bug

sprt committed:
Let's not declare useless variables

JuhQ commented:
Finally! :D
